lib/cli/exit-handler.js

```diff
@@ -57,6 +67,18 @@ class ExitHandler {
     this.#process.off('exit', this.#handleProcesExitAndReset)
     this.#process.off('uncaughtException', this.#handleExit)
     this.#process.off('unhandledRejection', this.#handleExit)
+
+    // Clean up signal handlers that were added for Docker/container support
```

Comment:
Is this really limited to docker/container support or is that just where we're most likely to see it?  I think this comment erroneously leads us to believe that's the only place we are expecting signals.